DEVELOPER_GUIDE.md

```diff
@@ -281,6 +282,18 @@ This method can also be used to develop against the [full distribution of OpenSe
 
 _This step is only mandatory if you have the [`security` plugin](https://github.com/opensearch-project/security) installed on your OpenSearch cluster with https/authentication enabled._
 
+> 1. Run `export initialAdminPassword=<initial admin password>` since it's needed by the configuration script
+> 2. Run `yarn opensearch snapshot --security`
+> 3. Wait a few seconds while the plugin is installed, configured, and OpenSearch starts up.
+
+Then within another window. You can start:
+
```

Comment:
Should we also add a step here to clone the dashboard-security repo under `/plugins`?